pavelib/prereqs.py

```diff
@@ -213,6 +213,11 @@ def uninstall_python_packages():
             sh("pip uninstall --disable-pip-version-check -y django-storages")
             uninstalled = True
 
+        # Uninstall django-oauth2-provider
+        if any("django-oauth2-provider==" in line for line in frozen):
+            sh("pip uninstall --disable-pip-version-check -y django-oauth2-provider")
+            uninstalled = True
+
```

Comment:
This won't run unless you also update `expected_version` on line 186.  Are you sure you need to perform an explicit uninstall?